### DISCUSSION TITLE
database encryption settings

### DISCUSSION THREAD

**Discussion by ms1222** [Kunzisoft/KeePassDX#1473] [Q&A]
I cannot open my file created with Keepass on a PC,  I copied the .kdbx file and the keyfile.  
Any suggestions (I plan on continuing using keepass on the PC and periodically copying the file to my Samsung tablet).

My database encryption algorithm is  AES/Rijndael(256-bit, FIPS 197
 and AES-KEF derivation function.  with a couple of thousand iterations.

Thanks in advance.

**Comment by J-Jamet** [comment]
Can you upload in this thread a newly created database that is compatible with KeePass and not compatible with KeePassDX, this will allow to see what is the problem.
